src/views/TerminologyView.vue

```diff
@@ -0,0 +1,344 @@
+      this.isDataLoading = true;
+
+      const apiHeaders = checkLogInAndAppendAuthHeaders({
+        "Content-Type": "application/json",
+      });
+      Promise.all([
+        fetch(`/gene2phenotype/api/attribs/description`, {
+          method: "GET",
+          headers: apiHeaders,
+        }),
+        fetch(`/gene2phenotype/api/molecular_mechanisms/`, {
+          method: "GET",
+          headers: apiHeaders,
+        }),
+        fetch(`/gene2phenotype/api/ontology_terms/variant_types/`, {
+          method: "GET",
+          headers: apiHeaders,
+        }),
+      ])
+        .then((responseArr) => {
+          return Promise.all(
+            responseArr.map((response) => {
+              if (response.status === 200) {
+                return response.json();
+              } else {
+                return Promise.reject(
+                  new Error("Unable to fetch descriptions data")
+                );
+              }
+            })
+          );
+        })
+        .then((responseJsonArr) => {
+          const [
+            terminologyDescriptionData,
+            molecularDescriptionData,
+            variantDescriptionData,
+          ] = responseJsonArr;
+          this.isDataLoading = false;
+          this.terminologyDescriptionData = terminologyDescriptionData;
+          this.molecularDescriptionData = molecularDescriptionData;
+          this.variantDescriptionData = variantDescriptionData;
+        })
+        .catch((error) => {
+          this.isDataLoading = false;
+          this.errorMsg = error.message;
+          console.log(error);
+        });
+    },
+  },
+};
+</script>
+<template>
+  <div class="container-fluid d-flex">
+    <div>
+      <div class="p-3 bg-light border-end" style="min-width: 200px">
+        <h5 class="text-dark fw-bold">Terminology</h5>
+        <ul class="nav flex-column">
+          <li class="nav-item">
+            <a class="nav-link text-dark" href="#g2p-confidence"
+              >G2P Confidence Category</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-dark" href="#allelic-requirement"
+              >Allelic Requirement</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-dark" href="#cross-cutting-modifier"
+              >Cross Cutting Modifier</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-dark" href="#molecular-mechanism"
+              >Molecular Mechanism</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-dark" href="#mechanism-synopsis"
+              >Molecular Mechanism Synopsis</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link text-dark" href="#variant-consequences"
+              >Variant Consequences</a
+            >
+          </li>
+        </ul>
+      </div>
+    </div>
```

Comment:
We can fix this navigation issue in a separate PR. For now, please remove this block of code.

Reply:
Done